app/workers/publishing_api_worker.rb

```diff
@@ -8,7 +8,12 @@ def perform(model_name, id, update_type = nil, locale=I18n.default_locale.to_s)
 
     I18n.with_locale(locale) do
       payload = presenter.as_json
-      send_item(payload, locale)
+
+      begin
+        send_item(payload, locale)
+      rescue => e
```

Comment:
Should be more explicit about the type of error being rescued here, especially as you check `e.code` in `handle_error`.